### DISCUSSION TITLE
Using the free tier for a small non-profit that accepts donations

### DISCUSSION THREAD

**Discussion by JHacin** [vercel/community#1614] [Help] [answered]
### Summary

Hi,

I'm working as a volunteer on a website for a small non-profit organization that deals with animal rescue and rehoming. One of the main points of the website will of course be to motivate people to donate for various causes, since this is pretty much the main source of income for this organization.

The website is pretty simple and doesn't really require anything beyond the free plan (+ the price of the Pro plan seems a bit much for my use-case) , but I'm not sure if this kind of thing falls under commercial use, since it is not driven by profit.

I've noticed that I can apply for a sponsorship, but honestly I'd just be happy with keeping the site on the free plan :)

Thanks in advance for any help.

**Comment by amyegan** [comment, marked as answer]
Hi Jan! You are completely fine accepting donations on a Hobby account. We don't consider that [commercial usage](https://vercel.com/docs/concepts/limits/fair-use-policy#commercial-usage). But feel free to apply for sponsorship if you feel you have outgrown the [Hobby plan limits](https://vercel.com/docs/concepts/limits/overview#).

Keep doing what you do! 🐶 🐱 🐦 🏠

**Reply by JHacin** [reply]
Great, thanks!
